backend/src/server/routes/v3/signup-router.ts

```diff
@@ -153,6 +187,13 @@ export const registerSignupRouter = async (server: FastifyZodProvider) => {
       if (!userAgent) throw new Error("user agent header is required");
       const appCfg = getConfig();
 
+      const serverCfg = await getServerCfg();
+      if (!serverCfg.allowSignUp) {
```

Comment:
As i was going through the code base, the database field `inviteOnlySignup` is not used anywhere in business logic. It's get updated in config but we are not using that field anywhere.

Then realized we don't need to have that field. Because when signup is disabled we can still allow  nvited signup. So the ui gets simplified to toggle version. 

Meaning we can just show like old one to allow or disallow signup and then in allowed mode provide the filter by domain like now

Reply:
Or am i missing something in where you have to introduce this db field

Reply:
Yeah,  good question

> signup is disabled we can still allow nvited signup

If the sign up disabled but allowing invite can be confusing for a large Organizations.
If the super admin choose to disabled signup but one of the project owner invited someone can be problematic.

That's the only reason it's there.

It makes to remove  if you want to let someone invite to org even, if the signup is disabled.